controllers/front/ProductController.php

```diff
@@ -334,6 +334,7 @@ protected function assignPriceAndTax()
 
         $quantity_discounts = SpecificPrice::getQuantityDiscounts($id_product, $id_shop, $id_currency, $id_country, $id_group, null, true, (int)$this->context->customer->id);
         foreach ($quantity_discounts as &$quantity_discount) {
+	    $quantity_discount['base_price'] = 0;
```

Comment:
This line is indented with three spaces instead of four.

Also, please wrap this line with an `isset()` check, just to be sure, like this:

```php
if (!isset($quantity_discount['base_price'])) {
    $quantity_discount['base_price'] = 0;
}
```